src/test/java/org/fundacionjala/sfdc/stepdefinition/OpportunitySteps.java

```diff
@@ -0,0 +1,120 @@
+package org.fundacionjala.sfdc.stepdefinition;
+
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.fundacionjala.sfdc.commons.PropertiesManager;
+import org.fundacionjala.sfdc.pageobjects.opportunities.SFOEdit;
+import org.fundacionjala.sfdc.pageobjects.opportunities.SFOHome;
+import org.fundacionjala.sfdc.pageobjects.opportunities.SFOMainPage;
+import org.fundacionjala.sfdc.pageobjects.SalesForceMainTabClassic;
+import org.testng.Assert;
+
+import java.util.Properties;
+
+/**
+ * Class for steps of createOpportunity.feature.
+ */
+public class OpportunitySteps {
+    private SFOHome opportunitiesHome;
+    private SFOEdit editOpportunity;
+    private SFOMainPage opportunityOptionsPage;
+    private Properties propertiesManager;
+
+    /**
+     * Given step.
+     */
+    @Given("^Go to opportunities section$")
+    public void goToOpportunitiesSection() {
+        SalesForceMainTabClassic salesForceMainTab;
+        salesForceMainTab = new SalesForceMainTabClassic("a.opportunityBlock", "li#AllTab_Tab");
+        salesForceMainTab.displayOptions();
+        salesForceMainTab.goToAccounts();
+    }
+
+    /**
+     * When step.
+     */
+    @When("^I fill the required fields in the new opportunity form$")
+    public void iFillTheRequiredFieldsInTheNewOpportunityForm() {
+        opportunitiesHome = new SFOHome();
+        editOpportunity = new SFOEdit();
+        propertiesManager = PropertiesManager.getInstance().getConfig();
+        opportunitiesHome.clickNewOpportunity();
+        final String opName = propertiesManager.getProperty("opportunityName");
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "opportunityName" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)